### ISSUE TITLE
Need advice 

### ISSUE THREAD

**Issue by Edakksm** [aelnouby/Text-to-Image-Synthesis#16]
Hi,

- We trained our model for 200 epochs.
- Ran predict as follows
       python runtime.py --inference --split 2
- Images got generated in 'results' folder
- But all the images generated were little off - please see below:

![a beautiful flower with outer whorl of prominent big white sepals and the corolla contains several t](https://user-images.githubusercontent.com/32203344/38450935-6d4b96de-39f4-11e8-8b72-1666658b647a.jpg)

Can you please tell us what could have gone wrong?

**Comment by sriharsha-sammeta** [comment]
Yup, I also got the same exact result for trying it out on the test data using the above command. And the image looks like that only. I hope that author helps us with some direction to resolve this. 

I am guessing that may be the command we used to run the test inference is long. Can the author please suggest the right command.

**Comment by sriharsha-sammeta** [comment]
Sorry. Found our mistake. We forgot to pass the pre_trained models path. Otherwise it would initialize weights randomly and use it to generate images, which makes sense.

The command should be:

`python runtime.py --inference --split 2 --pre_trained_disc checkpoints/disc_190.pth --pre_trained_gen checkpoints/gen_190.pth`

After this, I got really good images:

a cluster of flowers with white petals and no visible pistils
![a cluster of flowers with white petals and no visible pistils](https://user-images.githubusercontent.com/6239997/38460229-afeefb5e-3a83-11e8-869a-5815663d3d7b.jpg)


a flower large multiple layered slender petals of bright yellow with a short stamen.?
![a flower large multiple layered slender petals of bright yellow with a short stamen](https://user-images.githubusercontent.com/6239997/38460237-d002cfec-3a83-11e8-9941-304257bda966.jpg)


a flower with bright, tall, orange pointed petals and dark forest green pedicel.?
![a flower with bright tall orange pointed petals and dark forest green pedicel](https://user-images.githubusercontent.com/6239997/38460240-d9abcc10-3a83-11e8-9bf8-2d45abb21991.jpg)


Looks cool. Thanks author. Awesome work :) Now, we will try to implement the same on birds and COCO dataset as well as try using WGAN.


Similarly on birds data set, it would be 

`python runtime.py --inference --pre_trained_disc checkpoints/birds/disc_190.pth --pre_trained_gen checkpoints/birds/gen_190.pth --save_path birds --dataset birds --split 2`
